pages/common/ykman-config.md

```diff
@@ -4,14 +4,14 @@
 > Note: you can use `ykman info` to see currently enabled applications.
 > More information: <https://docs.yubico.com/software/yubikey/tools/ykman/Base_Commands.html#ykman-config-options-command-args>.
 
-- Enable an application over USB or NFC (`--enable` can be used multiple times to specify more applications):
+- Enable an application over USB or NFC (`-e` or `--enable` can be used multiple times to specify more applications):
 
-`ykman config {{usb|nfc}} --enable {{otp|u2f|fido2|oath|piv|openpgp|hsmauth}}`
+`ykman config {{usb|nfc}} {{[-e|--enable]}} {{otp|u2f|fido2|oath|piv|openpgp|hsmauth}}`
 
-- Disable an application over USB or NFC (`--disable` can be used multiple times to specify more applications):
+- Disable an application over USB or NFC (`-d` or `--disable` can be used multiple times to specify more applications):
```

Comment:
I've been doing only longform options in descriptions. Don't you think that should be enough?